docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Clustering/KMeans.tt

```diff
@@ -0,0 +1,29 @@
+﻿<#@ include file="Clustering.ttinclude"#>
+<#+
+string ClassName = "KMeans";
+            string Trainer = "KMeans";
+            string TrainerOptions = null;
+            string InlineTrainerOptions = "numberOfClusters: 2";
```

Comment:
>numberOfClusters: 2 [](start = 43, length = 19)

10

Reply:
Are you suggesting to use more clusters?

---
In reply to: [275080967](https://github.com/dotnet/machinelearning/pull/3317#discussion_r275080967) [](ancestors = 275080967)